### PR TITLE
Updating readme to use testflight to distribute the app

### PR DESCRIPTION
**Pull request by iBYarchitect**
## Summary
Readme update is required for users who will install the application using testflight.

## Changes
- `README` updated

## Checks to complete
- [x] The PR title has a conventional commit style (short and descriptive).
- [x] The PR description includes essential information that helps understand the purpose of the changes.
- [x] PR has assignee, reviewers, milestone, it is properly linked to the project and to the issue.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,30 @@ As a pet sitter, you can provide the service of walks using your own schedule, p
 
 ## Installation
 
+### For version 0.4.0 and above
```
Comment (kirpichonok) [inline README.md:31]: Let's add this in the future if it'll be actual.
```suggestion
### For version 0.4.0
```

File: README.md [hunk 2]
```diff
@@ -28,6 +28,30 @@ As a pet sitter, you can provide the service of walks using your own schedule, p
...
+
+1. Click [Here](https://apps.apple.com/us/app/testflight/id899247664?mt=8) to find the TestFlight application in the AppStore
+
+1. Click “Download” to install the TestFlight application.
+
+1. After installation, open the application. A TestFlight app notification appears asking for permission to send notifications. Click “Allow” to receive notifications about app updates.
+
+1. On the “Welcome” screen, click the “Continue” button. You can now install the Woof mobile app. Follow the link below: https://testflight.apple.com/join/mGNJXIwt
```
Comment (kirpichonok) [inline README.md:39]: nit: 
Maybe make it a bit simpler
```suggestion
1. Download and install TestFlight application. [Link]((https://apps.apple.com/us/app/testflight/id899247664?mt=8) on TestFlight on App Store.

2. Open TestFlight and allow to receive notifications about app updates.

3. On the “Welcome” screen, click the “Continue” button. You can now install the Woof mobile app.
```

File: README.md [hunk 3]
```diff
@@ -28,6 +28,30 @@ As a pet sitter, you can provide the service of walks using your own schedule, p
...
+
+1. After accepting the invitation, click “Install” to install the Woof application.
+
+1. You can then open the Woof application by clicking the “Open” button.
+
+1. Finally, you will see a screen with the application version and changelog; here we publish what has been updated in the current version. Click “Next” to continue.
+
+1. Click “Start Testing” to complete the installation process.
+
+1. You will see the “Welcome” screen of the Woof app. Choose role and start using the application.
```
Comment (NadzeyaShpakouskaya) [inline README.md:51]:
```suggestion
**Pre-requisites**: 
Installed `TestFlight`. If you don't how to install the `TestFlight` please follow the next steps:

 1. Click [here](ttps://apps.apple.com/us/app/testflight/id899247664?mt=8) to find the `TestFlight` application in the `AppStore`.

1. Click `Download` to install the `TestFlight` application.

1. After installation, open the application. A `TestFlight` notification appears asking for permission to send notifications. Click `Allow` to receive notifications about app updates.

1. On the `Welcome` screen, click the `Continue` button. 

**Installation process**:
1. You can now install the `Woof` mobile app following the link below: https://testflight.apple.com/join/mGNJXIwt

1. The invitation screen for the `Woof` application will appear with a description of the application. Click `Accept` to accept the invitation to our beta testing program.

1. After accepting the invitation, click `Install` to install the `Woof` application.

1. You can then open the `Woof` application by clicking the `Open` button.

1. Finally, you will see a screen with the application version and changelog, here we publish what has been updated in the current version. Click `Next` to continue.

1. Click `Start Testing` to complete the installation process.

1. Now you can use the `Woof` application. Choose role and enjoy the application.
```

File: README.md (hunk 2 above)
Comment (iBYarchitect) [reply README.md:39]: In my opinion a more detailed version of the installation description is more user-friendly.

File: README.md (hunk 1 above)
Comment (iBYarchitect) [reply README.md]: Fixed with 4f56140

**Review by NadzeyaShpakouskaya — CHANGES_REQUESTED** [review]
Well done. Please choose and use the short version `app` or full version `application` through the guide.
Consider some formatting improvements:

File: README.md (hunk 3 above)
Comment (iBYarchitect) [reply README.md]: Ok, fixed with 8009a03

**Review by NadzeyaShpakouskaya — APPROVED** [review]
LGTM to unblock merge

**Pull request merged**